src/framework/config.ts

```diff
@@ -13,14 +14,17 @@ import {
   IntegrationStep,
 } from '../framework/execution';
 
+export class IntegrationInvocationConfigNotFoundError extends IntegrationError {
+  constructor(message: string) {
+    super({
+      code: 'INVOCATION_CONFIG_NOT_FOUND',
+      message,
+    });
+  }
+}
+
 /**
- * Supports a convention over configuration approach
- * to building integrations.
- *
- * ./src/instanceConfigFields
- * ./src/steps/*
- * ./src/getStepStartStates
- * ./src/validateInvocation
+ * Loads integration invocation configuration.
  */
 export async function loadConfig(
```

Comment:
Hopefully we don't use `loadConfig` in the deployment host project. I could see how this might be used by the CLI but in deployed environment I would expect something like:

```typescript
const invocationConfig = require('@jupiterone/graph-project');
await runIntegration(invocationConfig); // <--- something else runs the integration using the invocation config
```

Reply:
Yep, we won't be using the `loadConfig` function deployed env with integrations switching over to the `index` file.